### ISSUE TITLE
LuLu appears to be accessing the webcam 🤔

### ISSUE THREAD

**Issue by p4checo** [objective-see/LuLu#94]
Hi 👋 

Thanks for an amazing project!

I've recently received an XFENCE (previously Little Flocker) alert informing me that LuLu (1.1.0) was trying to access the webcam. This happened when I'm on the Mac's login screen (i.e. prior to even logging in).

Took a screenshot of the alert:
<img width="494" alt="lulu_webcam_access" src="https://user-images.githubusercontent.com/1391324/46426762-6667af80-c737-11e8-839c-958ce5bfb09a.png">

It appears to try to access the following path:

`/System/Library/Frameworks/CoreMediaIO.framework/Versions/A/Resources/VDC.plugin/Contents/Info.plist`

Since I don't see any particular reason for LuLu to be trying to access the webcam, I am wondering if this is a false positive of XFENCE.

Can you help me clarify this? Thanks! 🙏

**Comment by JayBrown** [comment]
LuLu probably uses a system framework somewhere that (in principle) has access to the camera. Doesn't mean that LuLu is _actually_ accessing the camera. This also happens on some websites. You load a webpage, and suddenly XFENCE tells you about a webcam access, even if you have disabled the camera with a macOS Profile. I assume it's just macOS needing to use a framework, which has built-in camera support, to load/render that webpage. Could be similar (or the same) with LuLu. It's probably some `#include` thingy (IOKit maybe?).

**Comment by p4checo** [comment]
Thanks for the information!

I also think this is most likely a false positive due to a LuLu import, but at the same time can create some discomfort on users when this gets picked up by XFENCE, especially on the login screen.

If this is indeed a legitimate usage of that framework which is causing a false positive (and misinformation), then perhaps I can report this on the XFENCE bugtracker/forum.

Cheers! 🍻

**Comment by JayBrown** [comment]
Do they have a real bug tracker? I also have a couple of things to report. :)

**Comment by p4checo** [comment]
Great question! 😆 

Even though I've had Little Flocker for years and I've applied for the XFENCE "beta", I still haven't been approved...  Still, just went to the beta portal, and didn't find anything pointing to bug reporting either. 

Additionaly, some weeks ago I chatted with their support because of official Mojave support, and they simply pointed me to the forums, where I haven't received any response to my post yet.

So, in conclusion, I'm afraid our XFENCE reports won't have a bug tracker to be submitted to 😓

**Comment by p4checo** [comment]
Closing this, as this is most certainly a false positive of XFENCE.

Meanwhile, I got access to F-Secure's XFENCE beta program and reported this issue there.

Unfortunately I don't have my hopes high for this to get fixed anytime soon (as many other problems in XFENCE), as XFENCE seems to be quite abandoned. Not sure what F-Secure's plans are for it, but not the topic here anyway...

Sorry for any noise 🙏
